v6/emoji/emoji.py

```diff
@@ -37,13 +38,909 @@ class Plugin(RestExtension):
     def set_site(self, site):
         self.site = site
         roles.register_local_role('emoji', emoji_role)
+        if hasattr(site, 'transforms'):
+            site.transforms.append(Emojis)
+        else:
+            site.transforms = [Emojis]
 
 
 def emoji_role(name, rawtext, text, lineno, inliner, options={}, content=[]):
     text = text.lower()
+    LOGGER.warn('The role :emoji:`{0}` is deprecated. Use |{0}| instead'.format(text))
     node = nodes.image(
         uri='http://www.tortue.me/emoji/{0}.png'.format(text),
         alt=text,
         classes=['emoji'],
     )
     return [node], []
+
+
+class Emojis(Transform):
+    """
+    Replace some substitutions if they aren't defined in the document.
+    """
+    # run before the default Substitutions
+    default_priority = 210
+
+    # list from http://www.tortue.me/
+    emojis = set([
+                'bowtie',     # NOQA
```

Comment:
strictly, it's over-indented

On Fri, Apr 3, 2015 at 3:14 PM, Roberto Alsina <notifications@github.com>
wrote:

> In v6/emoji/emoji.py
> <https://github.com/getnikola/plugins/pull/73#discussion_r27745195>:
>
> >      node = nodes.image(
> >          uri='http://www.tortue.me/emoji/{0}.png'.format(text),
> >          alt=text,
> >          classes=['emoji'],
> >      )
> >      return [node], []
> > +
> > +
> > +class Emojis(Transform):
> > +    """
> > +    Replace some substitutions if they aren't defined in the document.
> > +    """
> > +    # run before the default Substitutions
> > +    default_priority = 210
> > +
> > +    # list from http://www.tortue.me/
> > +    emojis = set([
> > +                'bowtie',     # NOQA
>
> Just curious... why NOQA that one?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/getnikola/plugins/pull/73/files#r27745195>.
>



-- 
mgaitan.github.io
textosypretextos.com.ar <http://textosyprextextos.com.ar>